ui/app/templates/allocations/allocation/index.hbs

```diff
@@ -88,6 +88,8 @@
     </div>
   </div>
 
+  {{lifecycleChart taskStates=model.states}}
```

Comment:
I checked the branch out to play around and this component invocation breaks for me (expects `lifecycle-chart`) which makes me wonder why it works for you. Are we on different version of ember cli or something?

Reply:
`ember-cli: 3.12.1` for me, btw.

Reply:
wow, that’s a strange one. It’s true that I get `3.12.0` from `ember -v`. But I’m surprised the tests would pass on CircleCI too 🤔 

Regardless I didn’t use this style deliberately, so I’ve now changed it to the usual dasherised pattern, thanks.